frontend/scripts/react-components/shared/newsletter/newsletter-form.component.jsx

```diff
@@ -19,7 +20,8 @@ class NewsletterForm extends React.PureComponent {
         email: '',
         firstname: '',
         lastname: '',
-        organisation: ''
+        organisation: '',
+        country: ''
```

Comment:
@Bluesmile82 I think you need to add "country" here as well https://github.com/Vizzuality/trase/blob/develop/app/controllers/api/v3/newsletter_subscriptions_controller.rb#L13 

Then we should notify them about these changes just to make sure they catch them